docs/getstarted/getting-started.md

```diff
@@ -0,0 +1,277 @@
+
+Let's install a language extension to add support for Python, or any other programming language you are interested in.
+
+1. Select the **Extensions** view in the Activity Bar.
+
+    The Extensions view enables you to browse and install extensions from within VS Code.
+
+    ![Screenshot that shows the Extensions view, highlighting the Extensions icon in the Activity Bar.](images/getting-started/extensions-view.png)
+
+1. Enter *Python* in the Extension view search box to browse for Python-related extensions, select the **Python** extension published by Microsoft and select the **Install** button.
+
+    ![Screenshot that shows the Extensions view with a search for Python extensions.](images/getting-started/extensions-search-python.png)
+
+1. Now, create a new Python file `hello.py` in your workspace and start typing the following Python code:
+
+    ```python
+    def say_hello(name):
+        print("Hello, " + name)
+
+    say_hello("VS Code")
+    ```
+
+    Notice that you now get suggestions and IntelliSense for Python code.
+
+    ![Screen capture that shows IntelliSense in action for a Python file.](images/getting-started/python-intellisense.gif)
+
+Learn more about code [editing features](/docs/editor/codebasics.md), [IntelliSense](/docs/editor/intellisense.md), [code navigation](/docs/editor/editingevolved.md), and [refactoring](/docs/editor/refactoring.md) in VS Code.
+
+## Step 6: Run and debug your code
+
+One of the key features in VS Code is its great support for running and debugging code. VS Code has built-in support for running and debugging Node.js applications, and there are extensions available for many other programming languages, such as Python, Java, C#, and more.
+
+> **Note**: Make sure you have the necessary runtime and VS Code extensions installed on your computer. For example, to debug a Python program, you need to have Python installed on your computer.
+
+In this tutorial, you debug the `app.js` file to explore the built-in Node.js debugger. Most debugging concepts and features are applicable to other debuggers as well.
+
+Let's debug the `app.js` program by using the built-in Node.js debugger.
+
+1. In the `app.js` file, put the cursor on the `console.log` line and press `kbstyle(F9)` to set a breakpoint.
+
+    A red dot appears in the left margin of the editor, indicating that a breakpoint is set. With a breakpoint, you can pause the execution of your program at a specific line of code.
+
+    ![Screenshot that shows a breakpoint in the editor.](images/getting-started/javascript-set-breakpoint.png)
+
+1. Press `kbstyle(F5)` to start a debugging session, and then select the **Node.js** debugger from the list.
```

Comment:
Should we show the list?

Reply:
Actually, when the `app.js` file is open, `F5` will automatically pick the Node.js debugger.